student.rb

```diff
@@ -1,9 +1,12 @@
-require './person'
+require_relative 'person'
 # Define student
 class Student < Person
-  def initialize(classroom)
+  attr_accessor :name, :classroom
+
+  def initialize(_classroom)
     super()
-    @classroom = classroom
+    @name = name
+    @classroom = nil
```

Comment:
- [ ] Great job linking the classes. please kindly add a  method that serves as the 'belongs-to' side (a student belongs to a classroom). you can get more light to that [here](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships`)
eg
```ruby
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```

Reply:
Thanks for the keen observation @tufoinnkuo10  I've done that.